views/peeps/index.erb

```diff
@@ -0,0 +1,148 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Barlow+Condensed:wght@300;400&display=swap" rel="stylesheet">
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+  <title>chitter</title>
+  </head>
+  <style>
+
+
+  body {
+    margin: 1%;
+    padding: 0;
+    background-color: rgb(50, 128, 245);
+    font-family: 'Barlow Condensed', sans-serif;
+    
+  }
+  .nav {
+  display: flex;
```

Comment:
Your website looks amazing! Really well done on this as it looks great and has a lovely design to it too. Superb work!